src/env.h

```diff
@@ -120,248 +120,261 @@ constexpr size_t kFsStatsBufferLength = kFsStatsFieldsNumber * 2;
+  V(http2ping_constructor_template, v8::ObjectTemplate)                        \
+  V(http2settings_constructor_template, v8::ObjectTemplate)                    \
+  V(http2stream_constructor_template, v8::ObjectTemplate)                      \
+  V(immediate_callback_function, v8::Function)                                 \
+  V(inspector_console_api_object, v8::Object)                                  \
+  V(libuv_stream_wrap_ctor_template, v8::FunctionTemplate)                     \
+  V(message_port, v8::Object)                                                  \
+  V(message_port_constructor_template, v8::FunctionTemplate)                   \
+  V(native_modules_code_cache, v8::Object)                                     \
+  V(native_modules_code_cache_hash, v8::Object)                                \
+  V(native_module_column_offset, v8::Integer)                                  \
+  V(native_module_line_offset, v8::Integer)                                    \
+  V(native_module_parameters, v8::Array)                                       \
+  V(native_modules_source, v8::Object)                                         \
+  V(native_modules_source_hash, v8::Object)                                    \
+  V(native_modules_with_cache, v8::Set)                                        \
+  V(native_modules_without_cache, v8::Set)                                     \
+  V(pipe_constructor_template, v8::FunctionTemplate)                           \
+  V(performance_entry_callback, v8::Function)                                  \
+  V(performance_entry_template, v8::Function)                                  \
+  V(process_object, v8::Object)                                                \
+  V(promise_handler_function, v8::Function)                                    \
+  V(promise_wrap_template, v8::ObjectTemplate)                                 \
+  V(sab_lifetimepartner_constructor_template, v8::FunctionTemplate)            \
+  V(script_context_constructor_template, v8::FunctionTemplate)                 \
+  V(script_data_constructor_function, v8::Function)                            \
+  V(secure_context_constructor_template, v8::FunctionTemplate)                 \
+  V(shutdown_wrap_template, v8::ObjectTemplate)                                \
+  V(tcp_constructor_template, v8::FunctionTemplate)                            \
+  V(tick_callback_function, v8::Function)                                      \
+  V(timers_callback_function, v8::Function)                                    \
+  V(tls_wrap_constructor_function, v8::Function)                               \
+  V(trace_category_state_function, v8::Function)                               \
+  V(tty_constructor_template, v8::FunctionTemplate)                            \
+  V(udp_constructor_function, v8::Function)                                    \
+  V(url_constructor_function, v8::Function)                                    \
```

Comment:
Can we backport this part immediately after the PR lands, or split it into its own PR?

Reply:
I can try clang-format this in 10.x, not sure if it's necessary for 8.x